src/main/java/com/codeus/winter/config/DefaultBeanFactory.java

```diff
@@ -77,38 +110,56 @@ public Object createBean(Class<Object> beanClass)
         return newBean;
     }
 
+    /**
+     * Creating bean object with name.
+     * @param name bean's name
+     * @return bean object if its not possible throw exception.
+     */
     @Override
-    public Object createBean(String name) throws NotUniqueBeanDefinitionException {
+    public Object createBean(@Nonnull final String name) throws NotUniqueBeanDefinitionException {
         uniqueBeanChecking(name);
 
         Object newBean = new Object();
         singletonBeans.put(name, newBean);
         return newBean;
     }
 
+    /**
+     * Register bean in the bean's storage.
+     * @param name bean's name.
+     * @param beanDefinition bean's BeanDefinition.
+     * @param beanInstance bean's instance.
+     */
     @Override
-    public void registerBean(String name, BeanDefinition beanDefinition, Object beanInstance) {
+    public void registerBean(@Nonnull final String name,
+                             @Nonnull final BeanDefinition beanDefinition,
+                             @Nonnull final Object beanInstance) {
         if (beanDefinition.isSingleton()) {
             singletonBeans.put(name, beanInstance);
         }
         beanDefinitions.put(name, beanDefinition);
     }
 
+    /**
+     * Adding BeanPostProcessor to the storage.
+     *
+     * @param postProcessor BeanPostProcessor.
+     */
     @Override
-    public void addBeanPostProcessor(BeanPostProcessor postProcessor) {
+    public void addBeanPostProcessor(@Nonnull final BeanPostProcessor postProcessor) {
         postProcessors.add(postProcessor);
     }
 
-    private void uniqueBeanChecking(String beanName) {
-        var isNotUniq = singletonBeans.entrySet().stream()
-                .map(entry -> entry.getKey().equals(beanName))
-                .anyMatch(beanName::equals);
+    private void uniqueBeanChecking(@Nonnull final String beanName) {
+        var isNotUniq = singletonBeans.keySet().stream()
+                .map(o -> o.equals(beanName))
+                .anyMatch(t -> t.equals(true));
         if (isNotUniq) {
```

Comment:
To get rid of this if you can use .findAny().orElseThrow().
And, btw, if you choose to apply this option, prev comment wouldn't be relevant anymore